src/bun.js/modules/NodeModuleModule.cpp

```diff
@@ -628,6 +632,75 @@ static JSValue getGlobalPathsObject(VM& vm, JSObject* moduleObject)
         static_cast<ArrayAllocationProfile*>(nullptr), 0);
 }
 
+JSC_DEFINE_HOST_FUNCTION(jsFunctionSetCJSWrapperItem, (JSGlobalObject * globalObject, JSC::CallFrame* callFrame))
+{
+    auto scope = DECLARE_THROW_SCOPE(globalObject->vm());
+    JSValue a = callFrame->argument(0);
+    JSValue b = callFrame->argument(1);
+    Zig::GlobalObject* global = jsDynamicCast<Zig::GlobalObject*>(globalObject);
+    if (!global) return JSC::JSValue::encode(JSC::jsUndefined());
+    String aString = a.toWTFString(globalObject);
+    RETURN_IF_EXCEPTION(scope, {});
+    String bString = b.toWTFString(globalObject);
+    RETURN_IF_EXCEPTION(scope, {});
+    global->m_moduleWrapperStart = aString;
+    global->m_moduleWrapperEnd = bString;
+    global->hasOverriddenModuleWrapper = true;
+    return JSC::JSValue::encode(JSC::jsUndefined());
+}
+
+JSC_DEFINE_CUSTOM_GETTER(nodeModuleWrapper,
+    (JSGlobalObject * global,
+        EncodedJSValue thisValue,
+        PropertyName propertyName))
+{
+    // This does not cache anything because it is assumed nobody
```

Comment:
uses it?

Reply:
updated comment
    // This does not cache anything because it is assumed nobody reads it more than once.